modules/notifications/notifications.php

```diff
@@ -740,35 +745,41 @@ function notification_comment( $comment ) {
 		// @TODO: mention if it was a reply
 		/*
 		if($parent) {
-			
+
 		}
 		*/
-		
-		
+
+
 		$body .= "\r\n--------------------\r\n";
 		// Insert the notification list from comment meta @see EF_Editorial_Comments->maybe_output_comment_meta()
 		if ($notification_list) {
 			$body .= esc_html__( 'Notified', 'edit-flow' ) . ": " . esc_html( $notification_list ) . "\n";
 		}
-		
+
 		$edit_link = htmlspecialchars_decode( get_edit_post_link( $post_id ) );
 		$view_link = htmlspecialchars_decode( get_permalink( $post_id ) );
-		
+
 		$body .= "\r\n";
 		$body .= __( '== Actions ==', 'edit-flow' ) . "\r\n";
 		$body .= sprintf( __( 'Reply: %s', 'edit-flow' ), $edit_link . '#editorialcomments/reply/' . $comment->comment_ID ) . "\r\n";
 		$body .= sprintf( __( 'Add editorial comment: %s', 'edit-flow' ), $edit_link . '#editorialcomments/add' ) . "\r\n";
 		$body .= sprintf( __( 'Edit: %s', 'edit-flow' ), $edit_link ) . "\r\n";
 		$body .= sprintf( __( 'View: %s', 'edit-flow' ), $view_link ) . "\r\n";
-		
-		$body .= "\r\n" . sprintf( __( 'You can see all editorial comments on this %s here: ', 'edit-flow' ), $post_type ). "\r\n";		
+
+		$body .= "\r\n" . sprintf( __( 'You can see all editorial comments on this %s here: ', 'edit-flow' ), $post_type ). "\r\n";
 		$body .= $edit_link . "#editorialcomments" . "\r\n\r\n";
-		
+
 		$body .= $this->get_notification_footer($post);
-		
+
 		$this->send_email( 'comment', $post, $subject, $body );
+
+		$format = "*%s* left a comment on *%s #%s - %s*\n\n %s";
```

Comment:
We could add a handful of i18n changes here. Can we:

1. Update this to a format string ( `__( '...', 'edit-flow' )` )
2. Use parameter numbers for interpolation (`%1$s`)
3. Add a `translators:` string?

Here's an example from another place in the plugin doing all three:

https://github.com/Automattic/Edit-Flow/blob/b7d70af548a9aff8eeca24e1d0e0d5121e1b8d08/modules/notifications/notifications.php#L652-L653

If you're new to WordPress i18n functions, [check out this page too](https://codex.wordpress.org/I18n_for_WordPress_Developers#Placeholders). Thanks!

Reply:
Yes I was planning to add i18n functions as well it's just that these text is not finalised yet. If you've noticed, that text isn't even in use 😄

Reply:
True! 😄